modules/level-6/cm-3010-databases-advanced-data-techniques/README.md

```diff
@@ -73,7 +73,14 @@ One two-hour unseen written examination and coursework (Type I)
 #### Semantic Web
 
 - [Linked Data Engineering (Semantic Web) OpenHPI](https://www.youtube.com/playlist?list=PLoOmvuyo5UAfY6jb46jCpMoqb-dbVewxg). - Prof. Dr. Harald Sack
+- [Working with RDF Graphs in Graph Studio](https://apexapps.oracle.com/pls/apex/r/dbpm/livelabs/view-workshop?wid=3195) - Oracle LiveLabs 
 
 #### SQL
 
 - [The Art of PostgreSQL](https://theartofpostgresql.com/). - _"The book that teaches SQL to developers: Learn to replace thousands of lines of code with simple queries!"_
+
+#### XML
+- [Databases: Semistructured Data](https://www.edx.org/learn/relational-databases/stanford-university-databases-semistructured-data) - Stanford Online
+
+#### Document/JSON
+- [Introduction to MongoDB](https://learn.mongodb.com/learn/learning-path/introduction-to-mongodb) - MongoDB University
```

Comment:
- A blank line should go between the heading and the list.
- The heading `#### Document/JSON` should go before `#### :heart: Notes` to maintain alphabetical order.

Reply:
@sglavoie I don't know, don't you think it's better to group the external resources at the end? the `notes` are resources directly related to the course while the other resources are additional external suggestions and so I think it's fine that they are all at the end of the document, just my opinion.

Reply:
Yes, I think you bring a great point. There hasn't really been much of a clear distinction in most of the Markdown files but that's indeed something that should be done. Agreed!